docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -201,13 +201,14 @@ createTestCafe('localhost', 1337, 1338)
 Enables TestCafe to take screenshots of the tested webpages.
 
 ```text
-screenshots(path [, takeOnFails]) → this
+screenshots(path [, takeOnFails, pathPattern]) → this
 ```
 
 Parameter                  | Type    | Description                                                                   | Default
 -------------------------- | ------- | ----------------------------------------------------------------------------- | -------
-`path`                     | String  | The path to which the screenshots are saved.
+`path`                     | String  | The root path where the screenshots are saved. The relative path is specified by the default [path patterns](../command-line-interface.md#path-patterns) or by the method's `screenshotPathPattern` parameter.
 `takeOnFails`&#160;*(optional)* | Boolean | Specifies if screenshots should be taken automatically when a test fails. | `false`
+`sceenshotPathPattern`&#160;*(optional)* | String | The pattern to compose screenshot files' relative path and name. See [--screenshot-path-pattern](../command-line-interface.md#-p---screenshot-path-pattern) for information about available placeholders.
```

Comment:
See [--screenshot-path-pattern](../command-line-interface.md#-p---screenshot-path-pattern) for information about available placeholders. -> See [--screenshot-path-pattern](../command-line-interface.md#-p---screenshot-path-pattern) for information about **the** available placeholders.